plugins/connection/flightctl_console.py

```
@@ -0,0 +1,481 @@
+                    output += content
+
+                    # Only break if we received CMD_END_MARKER
+                    if CMD_END_MARKER in output:
+                        break
+                elif channel == 2:
+                    err_output += content
+                elif channel == 3:
+                    raise Exception("Stream error occurred: " + content)
+
+            return output.replace(CMD_END_MARKER, "").strip(), err_output.strip()
+        except (ConnectionClosedOK, ConnectionClosedError) as e:
```

Ruff (0.8.2)

Local variable `e` is assigned to but never used

Remove assignment to unused variable `e`

(F841)